client/src/pages/Home.js

```diff
@@ -26,16 +26,25 @@ export function Home() {
 	return (
 		<main role="main">
 			<div>
-				<img
+				{/* <img
 					className="logo"
 					data-qa="logo"
 					src={logo}
 					alt="Just the React logo"
 				/>
 				<h1 className="message" data-qa="message">
 					{message}
-				</h1>
-				<Link to="/about/this/site">About</Link>
+				</h1> */}
+				<div className="container ">
+				<ul className="list-group list-group-flush">
+					<li className="list-group-item list-group-item-action">
+						<Link to="/about/this/site">About</Link>
```

Comment:
We don't need an About page for our app so we can remove it. Can we delete the About page in this PR as well?

Reply:
Ok, working on that.